telemetry/specs/INT.mdk

```diff
@@ -435,6 +440,13 @@ is shared between multiple queues. The format of this 4-octet metadata field is
 implementation specific and the metadata semantics YANG model shall describe the format 
 and units of this metadata field in the metadata stack.
 
+* Buffer occupancy
+  - The build-up of traffic in the buffer (in bytes, cells, or packets) that the
+INT packet observes in the device while being forwarded. Use case is when the buffer 
+is shared between multiple queues. The format of this 4-octet metadata field is switch 
+implementation specific and the metadata semantics YANG model shall describe the format 
+and units of this metadata field in the metadata stack.
+
```

Comment:
The change above is still a duplicate and should be deleted.

Reply:
Deleted the duplicate paragraph.